node/cn/backend.go

```diff
@@ -522,6 +523,12 @@ func (s *CN) SetupKaiaxModules(ctx *node.ServiceContext, mValset valset.ValsetMo
 		mBuilder = builder_impl.NewBuilderModule()
 		mGasless = gasless_impl.NewGaslessModule()
 	)
+
+	var mGaslessMinBal *big.Int
+	if ctx.NodeType() == common.CONSENSUSNODE {
+		mGaslessMinBal = big.NewInt(1e18)
```

Comment:
How about setting this value in `gasless/imp/constant.go`?

Reply:
fixed 7c6b5687c